Projects/UOContent/Items/Containers/TreasureMapChest.cs

```diff
@@ -361,6 +361,11 @@ private bool CheckLoot(Mobile m, bool criminalAction)
 
     public override void OnItemLifted(Mobile from, Item item)
     {
+        if (_lifted == null)
```

Comment:
Let's change to this to:
```cs
_lifted ??= new HashSet<Item>();
```

And move it right above this line:
```cs
_lifted.Add(item);
```

Reply:
@kamronbatman I guess we need to keep it above 

`var notYetLifted = !_lifted.Contains(item);`

which is where the exception happened.